test/tests/kernels/2d_diffusion/tests

```diff
@@ -35,6 +56,9 @@
     cli_args = 'Executioner/nl_abs_tol=1e+1 Executioner/petsc_options_iname=-snes_force_iteration Executioner/petsc_options_value=1'
     petsc_version = '>=3.8.4'
     prereq = testdirichlet
+    requirement = 'MOOSE shall test if the solver will take at least one iteration regardless of the initial residual norm when snes_force_iteration option is provided.'
```

Comment:
MOOSE shall force the solver to take at least one iteration regardless of the initial residual norm when the snes_force_iteration option is specified.